### ISSUE TITLE
Fail when translating nested, select or plurals

### ISSUE THREAD

**Issue by ingfraga** [ivonederlof/xlf-translator#4]
how can i help you to support this important features?

<span i18n>The author is {gender, select, male {male} female {female} other {other}}</span>

  <source>{VAR_PLURAL, plural, =0 {just now} =1 {one minute ago} other {<x id="INTERPOLATION" equiv-text="{{minutes}}"/> minutes ago by {VAR_SELECT, select, male {male} female {female} other {other} }} }</source>

**Comment by ivonederlof** [comment]
You're right! It's not taken in account. I'll rewrite this project soon. Until then, thank you. Suggestions/contributions are welcome.
